tests/test_core.py

```diff
@@ -282,6 +287,44 @@ def test_fetch_attachment_content(self):
                 )
             )
 
+    def test_add_attachment_content(self):
+        """Tests add_attachment_content method success."""
+        # slims_api is dynamically added to slims client
+        mock_response = MagicMock()
+        mock_response.text.return_value = self.example_add_attachments_response_text
+        with patch.object(
```

Comment:
The other unit tests are patching things with the `@patch` decorator. We should stick with a single implementation.

Reply:
Addressed here:
2a5c2f7